modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * ingress/configure-ingress-operator.adoc
+// * ingress/ingress-operator.adoc
+
 :_mod-docs-content-type: REFERENCE
 [id="nw-ingress-controller-configuration-parameters_{context}"]
 = Ingress Controller configuration parameters
 
-The `ingresscontrollers.operator.openshift.io` resource offers the following
-configuration parameters.
+The `ingresscontrollers.operator.openshift.io` resource includes optional configuration parameters that you can configure to meet specific needs for your organization.
```

Comment:
> `ingresscontrollers.operator.openshift.io` resource

I see this is existing text. But, I'm wondering if we should change this to something in English.

The `Infrastructure` custom resource (CR), named `cluster`,  includes.......

Reply:
:heavy_check_mark: